### ISSUE TITLE
[BUG] app doesn’t show up on home screen but uses an app Id

### ISSUE THREAD

**Issue by Jamzespec** [SideStore/SideStore#720]
### Describe the bug

I tried side loading a Minecraft ipa and it was downloading and it kicks me out of the app and uses a app Id and Minecraft doesn’t show up on my Home Screen

### Instructions to reproduce

Download the ipa https://ipalite.com/minecraft/download/v1.21.31-ipa/ and sideload it

### What version of SideStore are you using?

0.5.8

**Comment by mahee96** [comment]
Yes this is working as expected, sidestore tries to install the app, if for any reason there is a failure the previous steps can’t be undone so a registered appID will stay registered until its expiry
